### DISCUSSION TITLE
Validation input type number

### DISCUSSION THREAD

**Discussion by alex1897** [livewire/livewire#5754] [1. Help]
I was wondering about the validation error I received in the frontend when adding an none numeric value:

<img width="615" alt="Bildschirmfoto 2023-07-05 um 17 05 51" src="https://github.com/livewire/livewire/assets/6780121/df84bad0-bfe1-4b83-b6dc-8a6242adfa0b">

```
<input type="number" wire:model="overtime" name="overtime"> 
```
That's my validation:
![image](https://github.com/livewire/livewire/assets/6780121/71ea0238-4a23-401d-86c2-4a9b2d5b51b8)

It seems that Livewire is clearing the value if its none numeric:
![image](https://github.com/livewire/livewire/assets/6780121/8e06f89b-b905-46af-8601-e6d650f2c5fe)

Is that a bug or the expected behavior? If yes, how can I archive to show the user a correct error message?

**Comment by MarioPerini** [comment]
On Form Inputs with type="number" you should not be able to input characters, but [Firefox as example has a bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1398528) which allows it.

If you look at the request Firefox does not POST the incorrect number so livewire has no chance to validate it
```JSON
{
	"updates": [
		{
			"payload": {
				"id": "vp38",
				"name": "number_field",
				"value": ""
			},
			"type": "syncInput"
		}
	]
}
```
